07-complex-data-type/src/ComplexDataTypeSample.java

```diff
@@ -0,0 +1,258 @@
+import com.terracottatech.store.*;
```

Comment:
In samples it would be a little friendlier to not use star imports so that people can see what is coming from where.